src/main/java/com/qunite/api/web/ResponseEntityControllerAdvice.java

```diff
@@ -49,7 +50,8 @@ public ResponseEntity<Map<String, String>> handleValidationError(
   @ExceptionHandler({
       ForbiddenAccessException.class,
       JWTDecodeException.class,
-      InvalidPasswordException.class})
+      InvalidPasswordException.class,
+      TokenExpiredException.class})
```

Comment:
I have not seen a place in your code where it could appear

Reply:
It`s an exception that is thrown by JWT.verify() method.